convert2rhel/actions/post_conversion/breadcrumbs_finish_collection.py

```diff
@@ -0,0 +1,33 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, breadcrumbs
+
+
+logger = logging.getLogger(__name__)
+
+
+class BreadcumbsFinishCollection(actions.Action):
+    id = "BREADCRUMBS_FINISH_COLLECTION"
```

Comment:
This should have a dependency in the end and be the last thing to run, we don't want to give a false report to the user about breadcrumbs if something fails before it

Reply:
Actually, we need to define the `dependencies` property in the `BreadcumbsFinishCollection` class itself. We will just need to make sure that it is being executed after the "last action". 

If we defined `BreadcumbsFinishCollection` as a dependency for every action, it means that all of them will depend on `BreadcumbsFinishCollection` executing first, which is not what we want. We want that `BreadcumbsFinishCollection` is executed as last thing possible

Reply:
Yeah, all or at least one that contains all. We want to make sure that this will be kinda of the latest thing to run (alongside with the https://github.com/oamg/convert2rhel/pull/1299)